### PR TITLE
8 discover section 27.03.2023

### LINKED ISSUE #8: Discover why millions of entrepreneurs chose us section
**As a user,**
I want to see a "Discover why millions of entrepreneurs chose us" section within "Let's see if we are a good fit." right after a "Choose a plan that matches your project" section.

### Links to design:
- [Figma web 1](https://www.figma.com/file/Pp48CPTdPVHY3Fkrxqn45b/Varbin-Tech_?node-id=466-6870&t=n7L1paaaJPB2wfjn-4)
- [Figma web 2](https://www.figma.com/file/Pp48CPTdPVHY3Fkrxqn45b/Varbin-Tech_?node-id=466-599&t=n7L1paaaJPB2wfjn-4)
- [Figma mob 1](https://www.figma.com/file/Pp48CPTdPVHY3Fkrxqn45b/Varbin-Tech_?node-id=597-7716&t=n7L1paaaJPB2wfjn-4)
- [Figma mob 2](https://www.figma.com/file/Pp48CPTdPVHY3Fkrxqn45b/Varbin-Tech_?node-id=598-13965&t=n7L1paaaJPB2wfjn-4)

### Requirements:
- It should be two separate components
- All icons in SVG format

### PR DESCRIPTION
**Pull request by InnaLazarenko**
### List of changes:
- resolved #8

### CODE REVIEW

File: components/common/navigation/NavigationLinks.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ const navigationRoutes = [
   { text: 'Case studies', href: '', scrollTo: '#caseStudies' },
   { text: 'Services', href: '', scrollTo: '#services' },
   { text: 'Pricing', href: '', scrollTo: '#pricing' },
-  { text: 'Blog', href: '', scrollTo: '' },
+  { text: 'Blog', href: '', scrollTo: '#blog' },
```
Comment (var-bin) [inline components/common/navigation/NavigationLinks.tsx:7]: Please, revert these changes

File: components/discover/Discover.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  PageContainer,
+  DiscoverTitle,
+  DiscoverColumn,
+  DiscoverItemContainer,
+  DiscoverItemTitle,
+} from './styled-components';
+
+const Discover = () => {
+  return (
+    <PageContainer id="blog">
```
Comment (var-bin) [inline components/discover/Discover.tsx:18]: Please, remove the `id` attribute. It shouldn't be blog. I am unsure how to call a link to this section

File: components/discover/styled-components.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+export const DiscoverItemContainer = styled('div')`
+  padding: 36px 0;
+`;
+
+export const DiscoverColumn = styled(Stack)(({ theme }) => ({
+  width: '50%',
+  borderTop: '2px solid',
+  borderColor: alpha(theme.palette.primary.main, 0.1),
+
+  '& > div': {
```
Comment (var-bin) [inline components/discover/styled-components.ts:39]: Is it possible to replace this selector with some CSS class?

**Review by var-bin — CHANGES_REQUESTED** [review]

File: components/common/navigation/NavigationLinks.tsx (hunk 1 above)
Comment (InnaLazarenko) [reply components/common/navigation/NavigationLinks.tsx:7]: '+

File: components/discover/Discover.tsx (hunk 2 above)
Comment (InnaLazarenko) [reply components/discover/Discover.tsx:18]: '+

File: components/questions/Questions.tsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            Flexible rendering and caching options, including Incremental Static Regeneration (ISR),
+            on a per-page level.
+          </Typography>
+        </Box>
+
+        {data.map(question => {
+          return (
+            <Accordion
+              key={question.id}
+              expanded={expanded === 'panel' + question.id}
```
[the remark this answers is not in the document]
Comment (InnaLazarenko) [reply components/questions/Questions.tsx:63]: '+

File: components/discover/styled-components.ts (hunk 3 above)
Comment (InnaLazarenko) [reply components/discover/styled-components.ts:39]: '+
